Dockerfile

```
@@ -1,11 +1,11 @@
-ARG APP_INSIGHTS_AGENT_VERSION=2.3.1
-FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.0
+ARG APP_INSIGHTS_AGENT_VERSION=2.5.1-BETA
```

beta shouldn't be needed?
```suggestion
ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
```